appinfo/info.xml

```diff
@@ -36,6 +36,9 @@
 	<dependencies>
 		<nextcloud min-version="20" max-version="24" />
 	</dependencies>
+	<background-jobs>
+		<job>OCA\Calendar\BackgroundJob\CleanupJob</job>
```

Comment:
Should be a better class/file name, to make it explicit that we're cleaning up outdated booking confirmations